gnovm/pkg/gnolang/internal/softfloat/gen/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func main() {
+	// File paths
+	goroot := os.Getenv("GOROOT")
+	softfloatSrc := fmt.Sprintf("%s/src/runtime/softfloat64.go", goroot)
+	softfloatTestSrc := fmt.Sprintf("%s/src/runtime/softfloat64_test.go", goroot)
+	softfloatDest := "runtime_softfloat64.go"
+	softfloatTestDest := "runtime_softfloat64_test.go"
+
+	// Process softfloat64.go file
+	processFile(softfloatSrc, softfloatDest, "runtime", "softfloat")
+
+	// Process softfloat64_test.go file
+	processTestFile(softfloatTestSrc, softfloatTestDest)
```

Comment:
This file is very specific anyway, I'd make this

```suggestion
	softfloatSrc := fmt.Sprintf("%s/src/runtime/softfloat64.go", goroot)
	softfloatTestSrc := fmt.Sprintf("%s/src/runtime/softfloat64_test.go", goroot)

	// Process softfloat64.go file
	processFile(softfloatSrc, "runtime_softfloat64.go")

	// Process softfloat64_test.go file
	processTestFile(softfloatTestSrc, "runtime_softfloat64_test.go")
```

Reply:
Note removing the other arguments from `processFile`. It is hardcoded in `processTestFile`, it makes no sense for it to be a parameter in `processFile`.

Reply:
Done here https://github.com/gnolang/gno/pull/3584/commits/b7f2a4d0913013b4d77895190febd5e993f96d72